### PR TITLE
[CORE-1947] Switch minuend and subtrahend

### PR DESCRIPTION
**Pull request by robert-uhl**
Subtract the lesser from the greater to get a positive integer.

### CODE REVIEW

**Review by jrockway — COMMENTED** [review]
This looks good, but there's another case:

```
src/server/pps/server/api_server.go
1815:           return errors.Errorf("Kubernetes name %q is %d characters longer than the %d max", k8sName, dnsLabelLimit-len(k8sName), dnsLabelLimit)
```

**Review by jrockway — APPROVED** [review]
Very nice cleanup!

File: src/pfs/pfs_test.go [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestProject_ValidateName(t *testing.T) {
```
Comment (brendoncarroll) [inline src/pfs/pfs_test.go:11]: Is it idiomatic to have underscores in test names?

**Review by brendoncarroll — APPROVED** [review]

File: src/pfs/pfs_test.go (hunk 1 above)
Comment (robert-uhl) [reply src/pfs/pfs_test.go:11]: I stole the pattern from [example names](https://pkg.go.dev/testing#hdr-Examples), which for methods are of the form `ExampleProject_ValidateName`. Granted, it is really weird to see an underscore in a name in Go.

**Pull request merged**